packages/ra-ui-materialui/src/form/SimpleFormIterator.js

```diff
@@ -62,8 +62,34 @@ const useStyles = makeStyles(
     { name: 'RaSimpleFormIterator' }
 );
 
+const DefaultAddButton = props => {
+    const { onClick = () => {}, className = '' } = props;
```

Comment:
I think you can remove this line and just spread the props on the button directly

Reply:
Yes, I can do that. In my opinion in this way, we have more visibility about the props passed. Also, we are assigning some default values for covering some random undefined or null cases.

I can still do that if you want.

Reply:
Empty function and empty class make no sense as default values

Reply:
alright, done.